packages/ui/src/components/SSelect/sizes-mixin.scss

```diff
@@ -0,0 +1,21 @@
+@mixin s-select-sizes {
+  &-sm {
+    height: 24px;
```

Comment:
It seems that these values could be useful in another components too. Maybe it's better to have size variables in common place and use it here as variables?

Reply:
The system of design variables are not finished yet. I guess that in the future each component will have its own parameters like `SelectHeightSm`, `SelectHeightMd` etc. Or maybe it will be configured via unified height variables. We don't know yet, so these sizes will be refactored soon anyway.